src/quick/framelessquickwindow.cpp

```diff
@@ -22,6 +22,14 @@
  * SOFTWARE.
  */
 
+#ifdef _MSC_VER
```

Comment:
The error come from QQuickWindow inside, so I have to put in here, It's the minimum scope I can put, I know it's urgly.
```
\Qt\vs2019_x64_5.15.2_install\include\QtNetwork\5.15.2\QtNetwork /DWIN32 /D_WINDOWS /EHsc /Zi /Ob0 /Od /RTC1 -MDd /utf-8 /W4 /WX -std:c++17 /showIncludes /Fosrc\quick\CMakeFiles\FramelessHelperQuick.dir\framelessquickwindow.cpp.obj /Fdsrc\quick\CMakeFiles\FramelessHelperQuick.dir\ /FS -c X:\dev\framelesshelper-2.0\src\quick\framelessquickwindow.cpp
X:\Qt\vs2019_x64_5.15.2_install\include\QtQuick\5.15.2\QtQuick\private\qquickwindow_p.h(250): error C2220: 以下警告被视为错误
X:\Qt\vs2019_x64_5.15.2_install\include\QtQuick\5.15.2\QtQuick\private\qquickwindow_p.h(250): warning C4458: “context”的声明隐藏了类成员
X:\Qt\vs2019_x64_5.15.2_install\include\QtQuick\5.15.2\QtQuick\private\qquickwindow_p.h(253): note: 参见“QQuickWindowPrivate::context”的声明
X:\Qt\vs2019_x64_5.15.2_install\include\QtQml\5.15.2\QtQml\private/qv4global_p.h(252): warning C4201: 使用了非标准扩展: 无名称的结构/联合
X:\Qt\vs2019_x64_5.15.2_install\include\QtQml\5.15.2\QtQml\private/qv4global_p.h(262): warning C4201: 使用了非标准扩展: 无名称的结构/联合
X:\Qt\vs2019_x64_5.15.2_install\include\QtQml\5.15.2\QtQml\private/qstringhash_p.h(213): warning C4458: “size”的声明隐藏了类成员
X:\Qt\vs2019_x64_5.15.2_install\include\QtQml\5.15.2\QtQml\private/qstringhash_p.h(175): note: 参见“QStringHashData::size”的声明
X:\Qt\vs2019_x64_5.15.2_install\include\QtQml\5.15.2\QtQml\private\qqmlnotifier_p.h(180): warning C4458: “sourceSignal”的声明隐藏了类成员
X:\Qt\vs2019_x64_5.15.2_install\include\QtQml\5.15.2\QtQml\private\qqmlnotifier_p.h(134): note: 参见“QQmlNotifierEndpoint::sourceSignal”的声明
X:\Qt\vs2019_x64_5.15.2_install\include\QtQml\5.15.2\QtQml\private/qv4compileddata_p.h(690): warning C4458: “nameIndex”的声明隐藏了类成员
X:\Qt\vs2019_x64_5.15.2_install\include\QtQml\5.15.2\QtQml\private/qv4compileddata_p.h(669): note: 参见“QV4::CompiledData::Property::nameIndex”的声明
X:\Qt\vs2019_x64_5.15.2_install\include\QtCore/qvarlengtharray.h(88): warning C4127: 条件表达式是常量
X:\Qt\vs2019_x64_5.15.2_install\include\QtCore/qvarlengtharray.h(88): note: 考虑改用 "if constexpr" 语句
X:\Qt\vs2019_x64_5.15.2_install\include\QtCore/qvarlengtharray.h(87): note: 在编译 类 模板 成员函数“QVarLengthArray<void *,256>::~QVarLengthArray(void)”时
X:\Qt\vs2019_x64_5.15.2_install\include\QtGui\5.15.2\QtGui\private/qtextengine_p.h(273): note: 查看对正在编译的函数 模板 实例化“QVarLengthArray<void *,256>::~QVarLengthArray(void)”的引用
X:\Qt\vs2019_x64_5.15.2_install\include\QtGui\5.15.2\QtGui\private/qtextengine_p.h(267): note: 查看对正在编译的 类 模板 实例化“QVarLengthArray<void *,256>”的引用
X:\Qt\vs2019_x64_5.15.2_install\include\QtCore/qvarlengtharray.h(288): warning C4127: 条件表达式是常量
X:\Qt\vs2019_x64_5.15.2_install\include\QtCore/qvarlengtharray.h(288): note: 考虑改用 "if constexpr" 语句
X:\Qt\vs2019_x64_5.15.2_install\include\QtCore/qvarlengtharray.h(277): note: 在编译 类 模板 成员函数“QVarLengthArray<void *,256>::QVarLengthArray(int)”时
X:\Qt\vs2019_x64_5.15.2_install\include\QtGui\5.15.2\QtGui\private/qtextengine_p.h(273): note: 查看对正在编译的函数 模板 实例化“QVarLengthArray<void *,256>::QVarLengthArray(int)”的引用
X:\Qt\vs2019_x64_5.15.2_install\include\QtCore/qvarlengtharray.h(395): warning C4127: 条件表达式是常量
X:\Qt\vs2019_x64_5.15.2_install\include\QtCore/qvarlengtharray.h(395): note: 考虑改用 "if constexpr" 语句
X:\Qt\vs2019_x64_5.15.2_install\include\QtCore/qvarlengtharray.h(376): note: 在编译 类 模板 成员函数“void QVarLengthArray<void *,256>::realloc(int,int)”时
X:\Qt\vs2019_x64_5.15.2_install\include\QtCore/qvarlengtharray.h(297): note: 查看对正在编译的函数 模板 实例化“void QVarLengthArray<void *,256>::realloc(int,int)”的引用
X:\Qt\vs2019_x64_5.15.2_install\include\QtCore/qvarlengtharray.h(277): note: 在编译 类 模板 成员函数“QVarLengthArray<void *,256>::QVarLengthArray(int)”时
X:\Qt\vs2019_x64_5.15.2_install\include\QtGui\5.15.2\QtGui\private/qtextengine_p.h(273): note: 查看对正在编译的函数 模板 实例化“QVarLengthArray<void *,256>::QVarLengthArray(int)”的引用
X:\Qt\vs2019_x64_5.15.2_install\include\QtCore/qvarlengtharray.h(418): warning C4127: 条件表达式是常量
X:\Qt\vs2019_x64_5.15.2_install\include\QtCore/qvarlengtharray.h(418): note: 考虑改用 "if constexpr" 语句
X:\Qt\vs2019_x64_5.15.2_install\include\QtCore/qvarlengtharray.h(427): warning C4127: 条件表达式是常量
X:\Qt\vs2019_x64_5.15.2_install\include\QtCore/qvarlengtharray.h(427): note: 考虑改用 "if constexpr" 语句
```